apps/studio/src/server/modules/site/site.service.ts

```diff
@@ -26,3 +26,30 @@ export const getSiteConfig = async (siteId: number) => {
     name,
   }
 }
+
+// Note: This overwrites the full site config
+// TODO: Should triger immediate re-publish of site
+export const setSiteConfig = async (
+  siteId: number,
+  config: IsomerSiteConfigProps,
+) => {
+  return db
+    .updateTable("Site")
+    .set({ config })
+    .where("id", "=", siteId)
+    .executeTakeFirstOrThrow()
+}
+
+// TODO: Should triger immediate re-publish of site
+export const setSiteNotification = async (
+  siteId: number,
+  notificationStr: string,
+) => {
+  return db
+    .updateTable("Site")
+    .set({
+      config: sql`jsonb_set(config, '{"notification"}', to_jsonb(${notificationStr}::text))`,
```

Comment:
Since we set the key in place, this might impact @karrui's PR that does the type gen for our JSON blobs. Could we check-in with him on this?

Reply:
its ok i can fix it up if it's affected. just merge first if you think its ready i can rebase easily with Graphite TM